PyInstaller/utils/hooks/__init__.py

```diff
@@ -1311,6 +1312,41 @@ def get_hook_config(hook_api, module_name, key):
     return value
 
 
+def include_or_exclude_file(filename, include_list=None, exclude_list=None):
```

Comment:
I know PyInstaller's public/private API scoping is already beyond helping but if we're going to put this in a public submodule with a public name and a nice pretty docstring then we ought to finish the job and add its stub to the API reference and add a news entry for it. Or if we don't want other people depending on it, give it an underscore prefixed name.

Reply:
Done.